Source/msgcmd.cpp

```diff
@@ -12,7 +12,16 @@
 struct EXTERNMESSAGE {
 	LIST_LINK(EXTERNMESSAGE) m_Link;
 	char command[COMMAND_LEN];

```

Comment:
Sorry for an offtopic, everyone.  But shouldn't this `struct EXTERNMESSAGE {` become a usual C++ class as defined in the header:
```
template <class T>
class TList {
public:
	TList();
	~TList();
	void UnlinkAll();
	void DeleteAll();
	T *Head();
```
..?

Reply:
There's very little difference between `struct` and `class`, they only different in regards to visibility of their members.

Reply:
I've heard about this earlier.  I just don't believe it was tried doing OOP via structs in the original code:)

Well, it's probably pull #933 or issue #266 which is more appropriate for the discussion.  Sorry for the noize.